DeepSpeech.py

```diff
@@ -659,19 +697,16 @@ def export():
     output_names_ops = [op.name for op in outputs.values() if isinstance(op, Operation)]
     output_names = ",".join(output_names_tensors + output_names_ops)
 
-    mapping = None
-    if FLAGS.export_tflite:
-        # Create a saver using variables from the above newly created graph
-        # Training graph uses LSTMFusedCell, but the TFLite inference graph uses
-        # a static RNN with a normal cell, so we need to rewrite the names to
-        # match the training weights when restoring.
-        def fixup(name):
-            if name.startswith('rnn/lstm_cell/'):
-                return name.replace('rnn/lstm_cell/', 'lstm_fused_cell/')
-            return name
-
-        mapping = {fixup(v.op.name): v for v in tf.global_variables()}
+    # Create a saver using variables from the above newly created graph
+    # Training graph uses LSTMFusedCell, but the TFLite inference graph uses
```

Comment:
nit: training can use CUDNN cells as well now

Reply:
Hm... Thinking more about this, I'm not sure this mapping is even needed anymore. Let me test it better.